main.go

```diff
@@ -13,7 +13,8 @@ func init() {
 }
 
 func main() {
-
+        
+	os.Setenv("GODEBUG", os.Getenv("GODEBUG")+",tls13=1")
```

Comment:
Hello @jmutkawoa,

Thanks for your contribution.
I was wondering what was the point of this line and if we need to merge it in the code base.

Regards,

Reply:
it's needed for golang 1.12 to enable tls 1.3:
https://golang.org/doc/go1.12#tls_1_3

Reply:
Thank you for the link.

I'm wondering why you have to retrieve the `GODEBUG` value in order to set it though.
What do you think of this instead:
 
```go
os.Setenv("GODEBUG", "tls13=1")
```